docs/docs/snippets/python_pkg.md

```diff
@@ -0,0 +1,25 @@
+- The following commands should be run after activating the Python environment where you plan to use `jupyter`.
+
+    - Install the `rsconnect-jupyter` package with the following command:
+    <div class="code-title">Terminal</div>
+    ```bash
+    pip install rsconnect_jupyter
```

Comment:
I like the dummy shell prefix for these (e.g. `$ pip install rsconnect_jupyter`), but it's not a must have.

Reply:
I like it as well, but I don't have a preference.